Only create Python runner when .py behaviors are present (#546)
hashintel/hash: packages/engine/README.md

        git checkout $latestTag
        ```

> - Python [3.7.x] is required if Python behaviors are present.

Reviewer: ```suggestion
- Python [3.7.x] is required for Python initialization or Python behaviors.
```

### Project Setup / Building

- Run `cargo build`
- Setup a Python environment by running `./lib/execution/src/runner/python/setup.sh` and follow the instructions from the help.

> **WIP** - In the future, setting up Python will only be required if Python behaviors are present, but for now it is always required.

> The initial setup time may be long at the moment. We need to use a fork of `rusty_v8` but expect significant improvement after [rusty_v8#926](https://github.com/denoland/rusty_v8/pull/926) is merged and a new crates.io release is made.

Author: I also removed this line as we use the release now.

> - If Python behaviors are present, you have to setup a Python environment by running `./lib/execution/src/runner/python/setup.sh` and follow the instructions from the help.

Author: I'm not sure about this, is there a better place?
